pwiz_tools/Skyline/TestTutorial/DiaTutorialTest.cs

```diff
@@ -327,7 +327,7 @@ protected override void DoTest()
             RestoreViewOnScreen(27);
             PauseForScreenShot<GraphChromatogram>("Chromatogram graph metafile", 26);
 
-            RunUI(SkylineWindow.SelectedNode.Expand);
+            RunUI(() => SkylineWindow.SelectedNode.Expand()); 

```

Comment:
This change seems irrelevant and is not changing what the test does. Probably should not be included.